util.py

```diff
@@ -23,15 +24,31 @@ def get_economics_status() -> Dict[str, Any]:
         status = {
             'latest_update': int(datetime.datetime.now().timestamp()),
             'backend': SERVER,
-            'total_supply': total_supply,
-            'circulating_supply': circulating_supply,
-            'mined_tokens': mined_tokens,
-            'released_premined_tokens': RELEASED_PREMINED_TOKENS
+            'total_supply':
+                get_decimals(total_supply, add_comma_separator) if decimals else total_supply,
```

Comment:
Why don't you create a `format_amount()` that receives decimals, etc? It seems easier than the if statement.

Reply:
Done